website/src/components/MenuTitleChildren/MenuTitleChildren.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react"
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import HoverVisible from "../hoverVisible/HoverVisible";
+import './MenuTitleChildren.scss';
+
+const RenderChild = (props) => {
+    const [isVisible, setIsVisible] = useState(false)
+    const selectedField = useSelector((state) => state.treeData.selectedField);
+    
+    return(
+            <div 
+                onMouseEnter={() => setIsVisible(true)}
+                onMouseLeave={() => setIsVisible(false)}
+                onMouseOver={() => setIsVisible(true)}
+                key={props.index} 
+                className={`child ${selectedField === props.path ? 'selected' : ''}`} 
+                onClick={() => props.onClick(props.path)}
+            >
+                <div className="child-name">
+                    {props.name}
+                </div>
+                {
+                    props.showDelete ?
+                        <HoverVisible isVisible={isVisible}>
+                            <div className="delete-img" onClick={props.onDeleteClick.bind(this, props.path)}>
+                                <i className="fa-solid fa-trash-can fa-sm"></i>
+                            </div>
+                        </HoverVisible>
+                    : ''
+                }
+                
+            </div>
+        
+    )
+}
+
+const MenuTitleChildren = (props) => {
+    const children = []
+    useEffect(()=>{}, [props.children])
+    props.children.forEach((child, index) => {
+        children.push(
+           <RenderChild key={index} path={child.path} name={child.name} index={index} onClick={props.onClick} onDeleteClick={props.onDeleteClick} showDelete={props.showDelete}/>
+        )
+    })
+
+    let renderChildren = ''
+    if(props.showChildren){
+        renderChildren = (children.length > 0 ? children : <div className={`child child-info`}>No entries in this section.</div>)
```

Comment:
if children are passed from props. so messages that describe this message should.
we can add a property called `noEntriesMessage`

That way we can also replace the message and reuse this components in many locations.